src/utils/ValidationUtils.ts

```diff
@@ -0,0 +1,278 @@
+import ITechRecord from "../../@Types/ITechRecord";
+import {VEHICLE_TYPE} from "../assets/Enums";
+
+export const vehicleType: string[] = [
+  "psv",
+  "trl",
+  "hgv"
+];
+
+export const fuelPropulsionSystem: string[] = [
+  "DieselPetrol",
+  "Hybrid",
+  "Electric",
+  "CNG",
+  "Fuel cell",
+  "LNG",
+  "Other"
+];
```

Comment:
I'd suggest to use and update the enums present in `src/Enums.ts` instead of adding them as util constants

Reply:
fixed in 10245